storage/mdadm/include/PURPOSE

```diff
@@ -0,0 +1,6 @@
+Please check http://wiki.test.redhat.com/fge/mdadm_include
+
+Add these lines into your Makefile:
+===
+@echo "RhtsRequires: rh-tests-kernel-storage-include" >> $(METADATA)
+@echo "RhtsRequires: rh-tests-kernel-storage-mdadm-include" >> $(METADATA)
```

Comment:
These libraries need to be open sourced, see the networking tests as an example including the metadata file which extracts the library, I would just remove this section completely: https://github.com/CKI-project/tests-beaker/blob/master/networking/socket/fuzz/metadata

Reply:
Since I don't see the lines in metadata in the PR them may not be needed at all?

Reply:
Yes, don't need this file, delete it